### PR TITLE
Task/limasetup

### PR DESCRIPTION
**Pull request by conbrad**
- Adds lima setup steps / mac m1 lima setup step
- Fixes doc link
# Test Links:
[Percentile Calculator](https://wps-pr-2404.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2404.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2404.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2404.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2404.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-2404.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-2404.apps.silver.devops.gov.bc.ca/hfi-calculator)
[FWI Calculator](https://wps-pr-2404.apps.silver.devops.gov.bc.ca/fwi-calculator)

### CODE REVIEW

File: api/README.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ You will need an environment file. See: `.env.example`. Contact current maintain
 
 - Automated (mac only):
```
Comment (andrea-williams) [inline api/README.md:7]: Could you add a note somewhere in this section reminding people to create the python alias in their bash profile? Some of us dolts will easily forget to do that.

Comment (andrea-williams) [reply api/README.md:7]: Please

**Review by andrea-williams — COMMENTED** [review]
Will try out these steps on my M1 pronto to test them out before approving. (Although if it works on your machine, it should work on mine...)

File: api/README.md (hunk 1 above)
Comment (conbrad) [reply api/README.md:7]: Done in: https://github.com/bcgov/wps/pull/2404/commits/c01c06ff92bb5ebe0e4a00764c771c03ae7496d9

File: api/README.md [hunk 2]
```diff
@@ -6,8 +6,10 @@ You will need an environment file. See: `.env.example`. Contact current maintain
...
   - Run `setup/mac.sh` in parent folder
-  - then `poetry_setup.sh`
+  - Follow [lima setup steps](../setup/LIMA.md)
+  - then run `poetry_setup.sh`
 - Manual (linux): See [manual setup](../docs/MANUAL_SETUP.md).
+  Note: you may want to alias `python3` as `python` in your profle
```
Comment (andrea-williams) [inline api/README.md:12]:
```suggestion
  Note: you may want to alias `python3` as `python` in your profile
```

**Review by andrea-williams — APPROVED** [review]
I didn't run lima.sh because I already had it installed on my M1. Followed the M1-specific instructions in LIMA.md and everything appeared to run fine. Haven't actually tried running Docker/lima yet though.

**Comment by dgboss** [comment]
I ran into an issue trying to run `nerdctl` - `nerdctl: command not found'
It looks like nerdctl isn't available when using the docker template

**Comment by conbrad** [comment]
> I ran into an issue trying to run `nerdctl` - `nerdctl: command not found` It looks like nerdctl isn't available when using the docker template.
> 
> Edit: Deleted the 'default' instance and created a new default with just `limactl start` and the rest of the M1 setup steps run fine.

Ok awesome, thanks @dgboss! I'll remove the line for running `nerdctl`

**Pull request merged**